StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -41,6 +41,12 @@ extension PaymentSheet {
         .multibanco,
     ]
 
+    /// An unordered list of paymentMethod types that can be saved in PaymentSheet
+    @_spi(STP) public static let supportedSavedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]
+
+    /// An unordered list of paymentMethod types that can be set as default in PaymentSheet when opted in to the "save as default" feature
```

Comment:
Same on the unordered bit, also maybe "paymentMethod types" -> `STPPaymentMethodType`.